core/creator/src/main/java/io/quarkus/creator/phase/augment/AugmentOutcome.java

```diff
@@ -38,4 +41,9 @@ public JarBuildItem getJar() {
     public NativeImageBuildItem getNativeImage() {
         return nativeImage;
     }
+
+    public DockerImageResultBuildItem getDockerimage() {
```

Comment:
I don't think we should be referring to docker explicitly. It should be a generic container image extension.